examples/spark2/pom.xml

```diff
@@ -62,6 +62,11 @@
       <version>${spark.version}</version>
       <scope>${spark.deps.scope}</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
+    </dependency>
```

Comment:
Is this dependency required? I don't see any special imports in example classes

Reply:
fixed